todo_app/app.py

```diff
@@ -1,15 +1,28 @@
-from flask import Flask
+from flask import Flask, render_template, request # used for jinja and getting post requests 
+
+from todo_app.flask_config import Config 
+
+from todo_app.data.session_items import * # can use * as a wildcard to take everything from another file
+
 
-from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+#display index.html template with list of items via get_items function
+@app.route('/', methods=['GET'])
 def index():
-    return 'Hello World!'
+    items_variable = get_items()
```

Comment:
Minor point and I assume it was just because you were getting used to python, but it should be clear that it's a variable so I'd say just `items` is a better name.